brave/browser-laptop
# Added BetterTTV extension. Fixes #4956

Test Plan:
- Load up the browser.
- Ensure BetterTTV is in `about:extensions`
- Visit https://twitch.tv.
- Ensure that BetterTTV is working:
  - Should have BetterTTV options (like dark mode) available and working
  - When logged in, should have access to BetterTTV emotes
